build/release.js

```diff
@@ -1,3 +1,4 @@
+var fs = require( "fs" );
```

Comment:
I make a distinction between modules that should be required when the exported function is called and ones that don't need to be.